_data/retail.yml

```diff
@@ -304,4 +314,3 @@ websites:
       tfa: No
       twitter: zazzle
       facebook: zazzle
-
```

Comment:
Please do not remove new lines at the end of files. They should be there in all of our `.yml` files. See our `.editorconfig` for our formatting rules. If you use an editor that supports plugins, you can look for and install the Editor Config plug-in.